eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -22,21 +16,7 @@ if ($ChangeLogLocation -and $VersionString)
 }
 else
 {
-  Import-Module (Join-Path $PSScriptRoot modules Package-Properties.psm1)
-  if ([System.String]::IsNullOrEmpty($Language))
-  {
-    if ($RepoName -match "azure-sdk-for-(?<lang>[^-]+)")
-    {
-      $Language = $matches["lang"]
-    }
-    else
-    {
-      Write-Error "Failed to set Language automatically. Please pass the appropriate Language as a parameter."
-      exit 1
-    }
-  }
-
-  $PackageProp = Get-PkgProperties -PackageName $PackageName -ServiceName $ServiceName -Language $Language -RepoRoot $RepoRoot
+  $PackageProp = Get-PkgProperties -PackageName $PackageName -ServiceName $ServiceName
   $validChangeLog = Confirm-ChangeLogEntry -ChangeLogLocation $PackageProp.pkgChangeLogPath -VersionString $PackageProp.pkgVersion -ForRelease $ForRelease
```

Comment:
One clean-up area I was wanted to see was in your PkgProperties type can you remove "pkg" from the front of all the properties? So instead of "pkgChangeLogPath" it would be "ChangeLogPath".